sympy/functions/special/bessel.py

```diff
@@ -2182,3 +2214,67 @@ def _eval_nseries(self, x, n, logx, cdir=0):
             f = self._eval_rewrite_as_intractable(*self.args)
             return f._eval_nseries(x, n, logx)
         return super()._eval_nseries(x, n, logx)
+
+
+class _airyais(Function):
+    """
+    Helper function to make the $\\mathrm{Airyai}(z)$
+    function tractable for the Gruntz algorithm.
+
+    """
+
+
+    def _eval_aseries(self, n, args0, x, logx):
+        from sympy.series.order import Order
+
+        point = args0[0]
+        if point in [S.Infinity, S.NegativeInfinity]:
+            z = self.args[0]
+            l = [gamma(k + Rational(5, 6))*gamma(k + Rational(1, 6))*\
+                 Rational(-3, 4)**k/(2*pi**2*factorial(k)*\
+                z**Rational(3*k + 1, 2)) for k in range(n)] + [Order(1/z**Rational(3*n + 1, 2), x)]
+            return (Add(*l))._eval_nseries(x, n, logx)
+
+        return super()._eval_aseries(n, args0, x, logx)
+
+    def _eval_rewrite_as_intractable(self, x):
+        return 2*airyai(x)*exp(2*x**Rational(3, 2)/3)/sqrt(pi*sqrt(x))
+
+    def _eval_nseries(self, x, n, logx, cdir=0):
+        x0 = self.args[0].limit(x, 0)
```

Comment:
Is it normal to compute a limit like this in an `_eval_nseries` method?

Reply:
Does the limit not need to potentially take account of the direction?

Reply:
> Is it normal to compute a limit like this in an _eval_nseries method?
Yes (has been done for quite some methods, should be fine)

> Does the limit not need to potentially take account of the direction?
Yes